store/wallet.ts

```diff
@@ -22,6 +38,29 @@ export const wallet = createSlice({
   reducers: {
     setStatus: (state, action: PayloadAction<WalletStatus>) => {
       state.status = action.payload
+    },
+    setTokens: (state, action: PayloadAction<AddressToken[]>) => {
+      state.tokens = action.payload
+    },
+    setUtxoBalance: (state, action: PayloadAction<string>) => {
+      state.utxoBalance = action.payload
     }
   }
 })
+
+export const { setStatus, setTokens, setUtxoBalance } = wallet.actions
```

Comment:
The reason why I didn't want to export this individually was it creates ambiguity, 

for example `setStatus` for what which action? To find out when reading and maintaining code you have to reference the imports to know what that code does. It's just one of those patterns that lead to code debt and introduces unnessary bugs.

While the long-form, `wallet.actions.setStatus` is clear and concise in what you are trying to achieve.

Reply:
Ok, only downside is we're importing the entire object.

Reply:
don't worry about this, it's automatically optimized when it's compiled re-`export/imports` is more or less for QOL purposes